parts/k8s/kubernetesbase.t

```diff
@@ -16,11 +16,6 @@
       {{range $index, $agent := .AgentPoolProfiles}}
           "{{.Name}}Index": {{$index}},
           {{template "k8s/kubernetesagentvars.t" .}}
-          {{if IsNSeriesSKU .}}
```

Comment:
This is cruft. I can't see any usage of this `registerWithGpuTaints` ARM variable anywhere.